### ISSUE TITLE
Redshift connection gets cast error in completion_refresh 

### ISSUE THREAD

**Issue by jklukas** [dbcli/pgcli#537]
I just did a `pip install -U pgcli` to get 1.0.0 and attempted to connect to Redshift. The shell opens to a prompt like expected:
```
Version: 1.0.0
Chat: https://gitter.im/dbcli/pgcli
Mail: https://groups.google.com/forum/#!forum/pgcli
Home: http://pgcli.com
analytics>
```

But within a few seconds, the following exception appears:
```
Traceback (most recent call last):
  File "/usr/local/Cellar/python/2.7.10_2/Frameworks/Python.framework/Versions/2.7/lib/python2.7/threading.py", line r
    self.run()
  File "/usr/local/Cellar/python/2.7.10_2/Frameworks/Python.framework/Versions/2.7/lib/python2.7/threading.py", line n
    self.__target(*self.__args, **self.__kwargs)
  File "/usr/local/lib/python2.7/site-packages/pgcli/completion_refresher.py", line 65, in _bg_refresh
    refresher(completer, executor)
  File "/usr/local/lib/python2.7/site-packages/pgcli/completion_refresher.py", line 105, in refresh_tables
    completer.extend_columns(executor.table_columns(), kind='tables')
  File "/usr/local/lib/python2.7/site-packages/pgcli/pgcompleter.py", line 142, in extend_columns
    for schema, relname, colname, datatype in column_data:
  File "/usr/local/lib/python2.7/site-packages/pgcli/pgexecute.py", line 364, in table_columns
    for row in self._columns(kinds=['r']):
  File "/usr/local/lib/python2.7/site-packages/pgcli/pgexecute.py", line 359, in _columns
    cur.execute(sql)
ProgrammingError: cannot cast type regtype to character varying
```

I can use the shell, but autocompletion of table names is not working.

**Comment by jklukas** [comment]
Testing this since Redshift was announced to be working in https://github.com/dbcli/pgcli/issues/282.

BTW, I'm very excited about the finally having a chance to use pgcli now that it supports Redshift.

**Comment by darikg** [comment]
Sorry! The oldest version of postgresql I've tested this on was 8.4, but it looks like Redshift is 8.0? Let me see if I can get it installed somewhere. I think I know how to fix it

**Comment by jklukas** [comment]
Yeah, Redshift is unfortunately 8.0.2-based, I believe. Glad to hear there's promise for finding a fix.

**Comment by darikg** [comment]
I can't figure out how to install pg8.0. I have what I think will fix it in #539, if someone could try that out (or somehow get me an account on a pg8.0 server) I'd appreciate it.

**Comment by amjith** [comment]
This is now fixed in the latest release v1.1.0
Please upgrade: 

```
pip install -U pgcli
```
